client/src/frontend/components/resource-player/Video/index.js

```diff
@@ -104,23 +104,49 @@ class ResourcePlayerVideo extends Component {
     this.props.dispatch(notificationActions.addNotification(notification));
   };
 
+  trackRef = el => {
+    if (!el) return;
+
+    el.addEventListener("load", () => {
+      if (!el.track?.cues) return;
+      Array.from(el.track.cues).forEach(c => {
+        // eslint-disable-next-line no-param-reassign
+        c.line = -3;
+      });
```

Comment:
What is this for? Might be good to add a comment for future developers.

Reply:
Is your question rhetorical? If it isn't, that moves the captions up from their default position, so the controls overlay doesn't cover them. I didn't know before I looked into this that they can't be positioned with css.

Reply:
It is! I mean, I didn't know that, but I don't think others will either, so a brief comment in your code would be good.

Reply:
Yeah, kind of interesting. `::cue` accepts only a limited set of css properties: https://developer.mozilla.org/en-US/docs/Web/CSS/::cue